Address Krysta's feedback on "Quickstart: Create a quantum-based random number generator in Azure Quantum" (#498)
MicrosoftDocs/quantum-docs: articles/includes/quickstart-qc-include-quantinuum.md

Next, we'll prepare your environment to run the program against the workspace you created.

> 1. From the Visual Studio Code menu, select **Terminal** -> **New Terminal**.

Reviewer: ```suggestion
1. From the Visual Studio Code menu, select **Terminal** > **New Terminal**.
```

   ```

    > [!NOTE]
    > The MyLocation parameter in the example above is the **Region** 
    > specified on the **Create Quantum Workspace** page when following 
    > the steps in [Create an Azure Quantum workspace](xref:microsoft.quantum.how-to.workspace).
    > Region and Location are synonymous.  The parameter value may be 

>     > expressed in mixed case surrounded by quotes, e.g. `-l "West US 2"`,
>     > or in lower case with no spaces or quotes, e.g. `-l westus2`.

Reviewer: ```suggestion
    > expressed in mixed case surrounded by quotes, for example, `-l "West US 2"`,
    > or in lower case with no spaces or quotes, such as `-l westus2`.
```
